pom.xml

```diff
@@ -47,7 +47,7 @@ under the License.
     </modules>
 
     <properties>
-        <flink.version>1.17.0</flink.version>
+        <flink.version>1.16.3</flink.version>
```

Comment:
We could even set it to 1.16.0 (since that's the lowest supported version), even though there shouldn't be any changes in a patch version of Flink

Reply:
I changed to 1.16.0, thanks
will merge after ci becomes green